app/src/main/java/com/nervousfish/nervousfish/exceptions/CannotHappenException.java

```diff
@@ -12,7 +12,7 @@
     private static final long serialVersionUID = -7030163248191198652L;
 
     /**
-     * Constructor for a new NoBluetoothException with a message.
+     * Constructor for a new CannotHappenException with a message.
```

Comment:
Yes, it is used. For example when the machine learning considers an interval neither short nor long